hypervisor/src/mshv/mod.rs

```diff
@@ -606,6 +608,56 @@ impl cpu::Vcpu for MshvVcpu {
 
                     Ok(cpu::VmExit::Ignore)
                 }
+                #[cfg(feature = "sev_snp")]
+                hv_message_type_HVMSG_GPA_ATTRIBUTE_INTERCEPT => {
+                    let info = x.to_gpa_attribute_info().unwrap();
+                    let host_vis = info.__bindgen_anon_1.host_visibility();
+                    if host_vis >= HV_MAP_GPA_READABLE | HV_MAP_GPA_WRITABLE {
+                        warn!("Ignored attribute intercept with full host visibility");
```

Comment:
This should be the case, i.e MSHV is wrongly doing a VM exit. Do you think warn is enough here?

Reply:
I think it is legit to let VM running, if device accidentally give access to same region which it has already provided access to. Ideally it is a bug device side so, I think warning should be good enough here. We should not kill the VM for this reason.